Console/UpgradeHelperCommand.php

```diff
@@ -37,6 +42,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $result['preferences'] = [];
         $result['overrides'] = [];
 
+        $output->writeln('Populating file override index...');
```

Comment:
This seems like debugging code. Is there a legitimate reason to keep this output?

Reply:
I'm fine with removing it. I added it since populating the index takes roughly a minute or so and I wanted to make it clear that something was still happening before the progress bar pops up

Reply:
Gotcha - Let's leave the message then.